frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -213,7 +213,7 @@ export const useNotebookRedirectLink = (): (() => Promise<string>) => {
       const call = (resolve: typeof presolve, reject: typeof preject) => {
         getRoute(notebookNamespace, routeName)
           .then((route) => {
-            resolve(`https://${route.spec.host}/notebook/${notebookNamespace}/${routeName}`);
+            resolve(`https://${route.spec.host}/notebook/${notebookNamespace}/${routeName}/`);
```

Comment:
this is curious -- why did this change happen? I assumed the route efforts was un-impacted by this new host information

Reply:
I saw the same thing but when i searched our code i found that other areas that create such a URL append `/`.

Reply:
The change here specifically is because of the way that the kf notebook-controller generates istio `VirtualService` resources  (function [here](https://github.com/opendatahub-io/kubeflow/blob/e5d179d9e8ecfea666be4fe0dc2f0d55505673e9/components/notebook-controller/controllers/notebook_controller.go#L516)) - essentially since the virtual service has the trailing `/` we have to add it here as well.

Reply:
Cameron and I talked and this is not needed for the ServiceMesh flow. We agreed this is not needed.